examples/vite/src/App.tsx

```diff
@@ -119,26 +127,25 @@ function App() {
   useEffect(() => {
     (async () => {
       if (pcdStr) {
-        console.log(pcdStr);
-        const pcd = await ZKEdDSAEventTicketPCDPackage.deserialize(
-          JSON.parse(pcdStr).pcd
-        );
+        setResult(pcdStr);
+        // console.log(pcdStr);
+        // const pcd = await ZKEdDSAEventTicketPCDPackage.deserialize(
+        //   JSON.parse(pcdStr).pcd
+        // );
 
-        if (pcd && (await ZKEdDSAEventTicketPCDPackage.verify(pcd))) {
-          setResult(
-            `Your email address is ${pcd.claim.partialTicket.attendeeEmail}`
-          );
-        } else {
-          setResult("Could not verify PCD");
-        }
+        // if (pcd && (await ZKEdDSAEventTicketPCDPackage.verify(pcd))) {
+        //   setResult(
+        //     `Your email address is ${pcd.claim.partialTicket.attendeeEmail}`
+        //   );
+        // } else {
+        //   setResult("Could not verify PCD");
+        // }
```

Comment:
should this be uncommented?

Reply:
I've replaced it with an explanatory comment